### DISCUSSION TITLE
Metadata with Speech To Text Audio

### DISCUSSION THREAD

**Discussion by prismvineeth** [deepgram/community#949] [General help] [answered]
Is there a way to pass some metadata with the speech transcription audio so that it comes back with the transcription ?

**Comment by deepgram-community** [comment, marked as answer]
:wave:  you should check out our docs on Tagging and using Extra Meta Data which might give you what you are looking for.

<https://developers.deepgram.com/docs/tagging>

<https://developers.deepgram.com/docs/extra-metadata>

> John Vajda
